apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog/GoogleStackdriverTarget.cs

```diff
@@ -488,6 +491,17 @@ private LogEntry BuildLogEntry(LogEventInfo loggingEvent)
                 Resource = _resource,
             };
 
+            if (!string.IsNullOrEmpty(_projectTraceId))
+            {
+                var traceId = RenderLogEvent(TraceId, loggingEvent);
+                if (!string.IsNullOrEmpty(traceId))
```

Comment:
Please add braces to the bodies of these single-statement `if` statements.

Reply:
+1 and everywhere else